<ui version="4.0" >
 <class>Tano</class>
 <widget class="QMainWindow" name="Tano" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tano Player</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/icons/images/tano.png</normaloff>:/icons/images/tano.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="centralLayout" >
    <item>
     <widget class="Video" native="1" name="videoWidget" />
    </item>
    <item>
     <widget class="QFrame" name="videoControlsFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="layoutButtons" >
         <item>
          <widget class="QLCDNumber" name="channelNumber" >
           <property name="minimumSize" >
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="numDigits" >
            <number>3</number>
           </property>
           <property name="segmentStyle" >
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0" >
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonPlay" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Play</string>
           </property>
           <property name="statusTip" >
            <string>Play</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/icons/images/player_play.png</normaloff>:/icons/images/player_play.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonStop" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Stop</string>
           </property>
           <property name="statusTip" >
            <string>Stop</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/icons/images/player_stop.png</normaloff>:/icons/images/player_stop.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonBack" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Back</string>
           </property>
           <property name="statusTip" >
            <string>Back</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/icons/images/player_rew.png</normaloff>:/icons/images/player_rew.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonNext" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Next</string>
           </property>
           <property name="statusTip" >
            <string>Next</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/icons/images/player_fwd.png</normaloff>:/icons/images/player_fwd.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRecord" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Record Now</string>
           </property>
           <property name="statusTip" >
            <string>Record Now</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/icons/images/record.png</normaloff>:/icons/images/record.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonMute" >
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Mute</string>
           </property>
           <property name="statusTip" >
            <string>Mute</string>
           </property>
           <property name="icon" >
            <iconset resource="images.qrc" >
             <normaloff>:/icons/images/player_mute.png</normaloff>:/icons/images/player_mute.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="volumeSlider" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum" >
            <number>200</number>
           </property>
           <property name="sliderPosition" >
            <number>50</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVolume" >
           <property name="minimumSize" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>50</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutDuration" >
         <item>
          <widget class="QLabel" name="labelDuration" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="durationSlider" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelLenght" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile" />
    <addaction name="actionOpenUrl" />
    <addaction name="actionOpen" />
    <addaction name="separator" />
    <addaction name="actionBrowser" />
    <addaction name="actionRecorder" />
    <addaction name="actionEpg" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="actionUpdate" />
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuShow" >
     <property name="title" >
      <string>Show / Hide</string>
     </property>
     <addaction name="actionToolbar" />
     <addaction name="actionChannel_info" />
    </widget>
    <addaction name="actionTop" />
    <addaction name="actionLite" />
    <addaction name="actionFullscreen" />
    <addaction name="menuShow" />
    <addaction name="separator" />
    <addaction name="actionEditPlaylist" />
    <addaction name="actionSettings" />
   </widget>
   <widget class="QMenu" name="menuMedia" >
    <property name="title" >
     <string>Media</string>
    </property>
    <widget class="QMenu" name="menuRatio" >
     <property name="title" >
      <string>Aspect ratio</string>
     </property>
     <property name="icon" >
      <iconset resource="images.qrc" >
       <normaloff>:/icons/images/video.png</normaloff>:/icons/images/video.png</iconset>
     </property>
     <addaction name="actionRatioOriginal" />
     <addaction name="actionRatio43" />
     <addaction name="actionRatio169" />
    </widget>
    <widget class="QMenu" name="menuVolume" >
     <property name="title" >
      <string>Volume</string>
     </property>
     <property name="icon" >
      <iconset resource="images.qrc" >
       <normaloff>:/icons/images/player_volume.png</normaloff>:/icons/images/player_volume.png</iconset>
     </property>
     <addaction name="actionMute" />
     <addaction name="actionVolumeUp" />
     <addaction name="actionVolumeDown" />
    </widget>
    <addaction name="actionPlay" />
    <addaction name="actionStop" />
    <addaction name="actionBack" />
    <addaction name="actionNext" />
    <addaction name="actionRecord" />
    <addaction name="menuVolume" />
    <addaction name="menuRatio" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuOptions" />
   <addaction name="menuMedia" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Main Toolbar</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenToolbar" />
   <addaction name="separator" />
   <addaction name="actionEditPlaylist" />
   <addaction name="separator" />
   <addaction name="actionBrowser" />
   <addaction name="actionEpg" />
   <addaction name="separator" />
   <addaction name="actionSettings" />
   <addaction name="actionRecorder" />
  </widget>
  <widget class="QDockWidget" name="playlistWidget" >
   <property name="windowIcon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/folder_video.png</normaloff>:/icons/images/folder_video.png</iconset>
   </property>
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="features" >
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Channel info</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout_4" >
     <item>
      <widget class="QLabel" name="labelLanguage" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNow" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNext" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRefresh" >
       <property name="text" >
        <string>Refresh EPG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="channelTabWidget" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabList" >
        <attribute name="title" >
         <string>Channel list</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <item>
          <widget class="QTreeWidget" name="playlistTree" >
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="animated" >
            <bool>true</bool>
           </property>
           <property name="columnCount" >
            <number>1</number>
           </property>
           <column>
            <property name="text" >
             <string>Channel</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEpg" >
        <attribute name="title" >
         <string>Today's Schedule</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5" >
         <item>
          <widget class="EpgToday" native="1" name="epgToday" />
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/tano.png</normaloff>:/icons/images/tano.png</iconset>
   </property>
   <property name="text" >
    <string>About Tano Player</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/settings.png</normaloff>:/icons/images/settings.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionFullscreen" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Channel list</string>
   </property>
  </action>
  <action name="actionOpenFile" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open &amp;File</string>
   </property>
  </action>
  <action name="actionBrowser" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/browser.png</normaloff>:/icons/images/browser.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Browser</string>
   </property>
  </action>
  <action name="actionEditPlaylist" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/edit.png</normaloff>:/icons/images/edit.png</iconset>
   </property>
   <property name="text" >
    <string>Edit Channel list</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/help.png</normaloff>:/icons/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionChannel_info" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/folder_video.png</normaloff>:/icons/images/folder_video.png</iconset>
   </property>
   <property name="text" >
    <string>Channel info</string>
   </property>
  </action>
  <action name="actionOpenUrl" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionUpdate" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/update.png</normaloff>:/icons/images/update.png</iconset>
   </property>
   <property name="text" >
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionEpg" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/siol.gif</normaloff>:/icons/images/siol.gif</iconset>
   </property>
   <property name="text" >
    <string>SiOL TV EPG</string>
   </property>
  </action>
  <action name="actionRatioOriginal" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Original</string>
   </property>
  </action>
  <action name="actionRatio43" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>4:3</string>
   </property>
  </action>
  <action name="actionRatio169" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>16:9</string>
   </property>
  </action>
  <action name="actionOpenToolbar" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionPlay" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_play.png</normaloff>:/icons/images/player_play.png</iconset>
   </property>
   <property name="text" >
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_stop.png</normaloff>:/icons/images/player_stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionTop" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Always on top</string>
   </property>
  </action>
  <action name="actionRestore" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/win.png</normaloff>:/icons/images/win.png</iconset>
   </property>
   <property name="text" >
    <string>Restore</string>
   </property>
  </action>
  <action name="actionLite" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Simple Mode</string>
   </property>
  </action>
  <action name="actionToolbar" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/toolbar.png</normaloff>:/icons/images/toolbar.png</iconset>
   </property>
   <property name="text" >
    <string>Main Toolbar</string>
   </property>
  </action>
  <action name="actionNext" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_fwd.png</normaloff>:/icons/images/player_fwd.png</iconset>
   </property>
   <property name="text" >
    <string>Next</string>
   </property>
  </action>
  <action name="actionBack" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_rew.png</normaloff>:/icons/images/player_rew.png</iconset>
   </property>
   <property name="text" >
    <string>Back</string>
   </property>
  </action>
  <action name="actionMute" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_mute.png</normaloff>:/icons/images/player_mute.png</iconset>
   </property>
   <property name="text" >
    <string>Mute</string>
   </property>
  </action>
  <action name="actionVolumeUp" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/up.png</normaloff>:/icons/images/up.png</iconset>
   </property>
   <property name="text" >
    <string>Volume up</string>
   </property>
  </action>
  <action name="actionVolumeDown" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/down.png</normaloff>:/icons/images/down.png</iconset>
   </property>
   <property name="text" >
    <string>Volume down</string>
   </property>
  </action>
  <action name="actionRecorder" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/recorder.png</normaloff>:/icons/images/recorder.png</iconset>
   </property>
   <property name="text" >
    <string>Recorder</string>
   </property>
  </action>
  <action name="actionRecord" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/record.png</normaloff>:/icons/images/record.png</iconset>
   </property>
   <property name="text" >
    <string>Record Now</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Video</class>
   <extends>QWidget</extends>
   <header>video/Video.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EpgToday</class>
   <extends>QWidget</extends>
   <header>epg/EpgToday.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelVolume</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>413</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>486</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>actionBack</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>196</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMute</sender>
   <signal>clicked()</signal>
   <receiver>actionMute</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>328</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonNext</sender>
   <signal>clicked()</signal>
   <receiver>actionNext</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>240</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRecord</sender>
   <signal>clicked()</signal>
   <receiver>actionRecord</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>284</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPlay</sender>
   <signal>clicked()</signal>
   <receiver>actionPlay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>108</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStop</sender>
   <signal>clicked()</signal>
   <receiver>actionStop</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>152</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
